app/src/main/java/com/bogdan/codeforceswatcher/features/problems/redux/requests/ProblemsRequests.kt

```diff
@@ -40,7 +37,10 @@ class ProblemsRequests {
                     var problems = listOf<Problem>()
                     withContext(Dispatchers.IO) {
                         problems = mergeProblems(problemsEn, problemsRu)
-                        bindProblemsToContests(problems)
+                        for (i in 0..3) {
```

Comment:
Oh, please, don't make things like this ever 🙏Add `delay` if request isn't initiated by user, for example.

Reply:
)))